contrib/opencensus-ext-ocagent/setup.py

```diff
@@ -41,6 +41,7 @@
     install_requires=[
         'grpcio >= 1.0.0, < 2.0.0',
         'opencensus >= 0.4.dev0, < 1.0.0',
+        'opencensus-proto >= 0.0.dev0, < 1.0.0',
```

Comment:
FYI we've been replacing "dev" in the release branches. If `opencensus-proto` follows the same versioning strategy the first release will be `0.1.0` and the version number on master after the release should be `0.2.dev0`, in which case this should be `0.1.dev0`.